ui/tests/acceptance/mfa-login-enforcement-test.js

```diff
@@ -134,6 +134,7 @@ module('Acceptance | mfa-login-enforcement', function (hooks) {
     assert.dom('h1').includesText(enforcement.name, 'Name renders in title');
     assert.dom('h1 svg').hasClass('flight-icon-lock', 'Lock icon renders in title');
     assert.dom('[data-test-tab="targets"]').hasClass('active', 'Targets tab is active by default');
+    await waitFor('[data-test-target]', { timeout: 5000 });
```

Comment:
This is strange that we need to do this, and I'm wondering if it's because we're using `@model` in the route template, which is known to cause some issues. Maybe try using `this.model` in `app/templates/vault/cluster/access/mfa/enforcements/enforcement/index.hbs` and removing this?

Reply:
Sauce: https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-model-argument-in-route-templates.md

Reply:
@chelshaw no luck. Though I did learn something new. Removing the waitFors I always hit the timeout issues even with the change you suggested.